src/store/actions/getNFTAssets.js

```diff
@@ -0,0 +1,38 @@
+export const getNFTAssets = async ({ getters, commit, state }, args) => {
+  const { network, walletId } = args
+  const client = getters.client({
+    network,
+    walletId,
+    asset: 'ETH'
+  })
+
+  const nft = await client.nft.fetch()
+
+  nft.assets.forEach((asset) => {
+    if (state.starredNFTs) {
+      asset['starred'] = state.starredNFTs.filter(
+        (item) =>
+          item.asset_contract.address === asset.asset_contract.address && item.id === asset.id
+      ).length
+        ? true
+        : false
+    } else {
+      asset['starred'] = false
+    }
+  })
+
+  commit('SET_NFT_ASSETS_NUMBER', nft.assets.length)
```

Comment:
There's no need to commit something to the state that can be derived from the state directly.
When you need the length you can do state.nfts.length